canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -154,7 +154,6 @@ class ToolTip extends React.Component {
 	}
 
 	showTooltipWithDelay() {
-
```

Comment:
Unless you have a really good reason, you should avoid removing (or adding) empty lines in the code you are not altering as part of your PR.